dashboard/test/controllers/openai_sessions_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class OpenaiSessionsControllerTest < ActionController::TestCase
+  include OpenaiChatHelper
+
+  setup do
+    response = Net::HTTPResponse.new(nil, '200', nil)
+    OpenaiChatHelper.stubs(:request_chat_completion).returns(response)
+    OpenaiChatHelper.stubs(:get_chat_completion_response_message).returns({})
+  end
+
+  # User without ai_chat_access is unable to access the chat completion endpoint
+  test_user_gets_response_for :chat_completion,
+  user: :levelbuilder,
```

Comment:
Nit: indent these arguments (I guess this isn't explicitly required by linting but I think more readable?)